common/parameters/Options.java

```diff
@@ -99,6 +101,17 @@ public SELF schemaLockAcquireTimeoutMillis(int acquireSchemaLockTimeoutMillis) {
         return getThis();
     }
 
+    public boolean primaryReplica() {
```

Comment:
I'm adding the replica options here since I cannot subclass the `Options` class in Grakn Cluster. This isn't ideal though and I'm open to ideas on how to solve the issue.